### ISSUE TITLE
Fix the error messages

### ISSUE THREAD

**Issue by xescugc** [cycloidio/cycloid-cli#224]
Currently any error message from the API looks like this on the CLI

```
[GET /organizations/{organization_canonical}/service_catalogs/{service_catalog_ref}][404] getServiceCatalogNotFound  &{Errors:[0xc00045c0f0]}
```

Which is a PITA because you actually don't know anything about the error, just that it was a `404` in this case (an easy one) but the actual message from the error is behind `[0xc00045c0f0]`.

The idea would that all the errors should pass through a parser that converts them to something more user readable, something like:

`A 404 error was returned on 'getServiceCatalogNotFound' call with message: "The Organization was not found"`

Basically a template with:

`A [STATUS_CODE] error was returned on [API_ACTION] call with message: [MESSAGE FROM THE ERROR]`

I don't think the URL is relevant as the `API_ACTION` is already that but user readable.

This will require a bit of refactoring but it'll improve a lot any usage/integration.
